.gitignore

```diff
@@ -40,4 +40,5 @@ logs/
 
 .nfs*
 pom.xml
+application.properties
```

Comment:
application.properties should not be part of gitignore, 
This files needs to change often.

Same goes for pom.xml